components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -367,33 +383,63 @@ const PluginItem = ({
     );
 };
 
-PluginItem.propTypes = {
-    pluginStatus: PropTypes.object.isRequired,
-    removing: PropTypes.bool.isRequired,
-    handleEnable: PropTypes.func.isRequired,
-    handleDisable: PropTypes.func.isRequired,
-    handleRemove: PropTypes.func.isRequired,
-    showInstances: PropTypes.bool.isRequired,
-    hasSettings: PropTypes.bool.isRequired,
+type Props = BaseProps & {
+    config: {
```

Comment:
I would extract this to a separate Interface especially if we pluginSettings can come from somewhere else and we would need to share that type.

Reply:
We actually have a type for the `Config` that they can use in `mattermost-redux`

Reply:
@sowmiyamuthuraman thanks for handling. Have you had a chance to consider my comment here?

Reply:
> I would extract this to a separate Interface especially if we pluginSettings can come from somewhere else and we would need to share that type.

@sowmiyamuthuraman you can still extract this to a separate interface if keys are missing, so we can reuse and share the type :)

Reply:
@nevyangelova I have made the changes. But the build is failing due to`sh: 1: eslint: not found`. can you pls help?